src/main.cc

```diff
@@ -1,5 +1,26 @@
+#include <iostream>
 #include "main.h"
 
-int main () {
-	return 0;
+int main() {
+
+  //let the player pick the ruleset
+  std::cout << "Which ruleset would you like to use? \n  0: Capture\n  1: Avalanche"<<std::endl;
+  int ruleset;
+  std::cin >> ruleset;
+
+  std::cout << "Starting Game..."<<std::endl;
+
+  mancala::Board game; // im not sure how you would initialize the ruleset
+
+  //game loop
+  bool endGame = false;
+  while(endGame == false) {
+    //ask player which # to move
```

Comment:
I like what I see. If anything I would just add the end condition, but I think we can do that later.